images/macos/software-report/SoftwareReport.Browsers.psm1

```diff
@@ -1,64 +1,72 @@
-function Get-BrowserSection {
-    return New-MDList -Style Unordered -Lines @(
-        (Get-SafariVersion),
-        (Get-SafariDriverVersion),
-        (Get-ChromeVersion),
-        (Get-ChromeDriverVersion),
-        (Get-EdgeVersion),
-        (Get-EdgeDriverVersion),
-        (Get-FirefoxVersion),
-        (Get-GeckodriverVersion),
-        (Get-SeleniumVersion)
+function Add-BrowserSection {
+    param (
+        [HeaderNode] $HeaderNode
     )
+
+    $HeaderNode.AddToolNode("Safari", $(Get-SafariVersion))
+    $HeaderNode.AddToolNode("SafariDriver", $(Get-SafariDriverVersion))
+    $HeaderNode.AddToolNode("Google Chrome", $(Get-ChromeVersion))
+    $HeaderNode.AddToolNode("ChromeDriver", $(Get-ChromeDriverVersion))
+    $HeaderNode.AddToolNode("Microsoft Edge", $(Get-EdgeVersion))
+    $HeaderNode.AddToolNode("Microsoft Edge WebDriver", $(Get-EdgeDriverVersion))
+    $HeaderNode.AddToolNode("Mozilla Firefox", $(Get-FirefoxVersion))
+    $HeaderNode.AddToolNode("geckodriver", $(Get-GeckodriverVersion))
+    $HeaderNode.AddToolNode("Selenium server", $(Get-SeleniumVersion))
 }
 
 function Get-SafariVersion {
     $version = Run-Command "defaults read /Applications/Safari.app/Contents/Info CFBundleShortVersionString"
     $build = Run-Command "defaults read /Applications/Safari.app/Contents/Info CFBundleVersion"
-    "Safari $version ($build)"
+    return "$version ($build)"
 }
 
 function Get-SafariDriverVersion {
     $version = Run-Command "safaridriver --version" | Take-Part -Part 3,4
-    "SafariDriver $version"
+    return $version
 }
 
 function Get-ChromeVersion {
     $chromePath = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
-    return Run-Command "'${chromePath}' --version"
+    $version = Run-Command "'${chromePath}' --version"
+    return $version.Substring("Google Chrome".Length).Trim()
```

Comment:
IMHO, `Substring` approach is not ideal because if output of tool is changed, we will just lose the part of output. Also, I prefer avoid hardcoding `Google Chrome` here.

I see two alternatives:
- Use `Take-Part`: `$version | Take-Part -Part 3`. With this approach, you also can lose some output if tool output is changed but at least we don't have hardcoded `Google Chrome`
- Use `TrimStart`: `$version.TrimStart("Google Chrome")`. With this approach it will truncate substring only in case if line is started with `Google Chrome`. With this approach we will have hardcoded string but we won't lose part of output accidentally.

The same comment for other functions with the same logic.

Reply:
I followed the second suggestion here